### PR TITLE
DOCSP-37554 Retrieve Distinct Values

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37554
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/pymongo/DOCSP-37554-distinct/read/distinct/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/distinct.txt [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   :values: reference
+ 
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve a list of distinct values across a collection for a
+specified field with {+driver-short+}.
```
Comment (mongoKart) [inline source/read/distinct.txt:24]: Thought: Would this be easier to parse as multiple sentences? Something like:

```suggestion
Within a collection, different documents might contain different values for a single field. 
For example, one ``restaurant`` document has a ``borough`` value of ``"Manhattan"``, and
another has a ``borough`` of ``"Queens"``.

With {+driver-short+}, you can retrieve all the distinct values
that a field contains across multiple documents.
```

If you like this format, you could do something similar in the subsections, like "We need to find all Italian restaurants in Manhattan" or something

File: source/read/distinct.txt [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      :end-before: end-distinct
+      :language:  python
+
+   .. output:: 
+
+      <placeholder output>
+
+The results show each distinct value that appears in the ``borough`` field
+across all documents in the collection. Although several documents have the same
+value in the ``borough`` field, each value only appears in the results once.
```
Comment (mongoKart) [inline source/read/distinct.txt:59]:
```suggestion
value in the ``borough`` field, each value appears in the results only once.
```

File: source/read/distinct.txt [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``filter``
+     - | A query filter that specifies which documents to retrieve distinct
```
Comment (mongoKart) [inline source/read/distinct.txt:102]:
```suggestion
     - | A query filter that specifies the documents to retrieve distinct
```

**Review by mongoKart — APPROVED** [review]
LGTM. Left a couple of suggestions

File: source/read/distinct.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/read/distinct.txt]: Yeah I like that, I was struggling to find a good way to phrase it in a short concise sentence. I changed the overview section to this and left the subsections as-is

**Pull request merged**